stdlib/3/wsgiref/types.pyi

```diff
@@ -0,0 +1,26 @@
+# Type declaration for a WSGI Function in Python 3
+#
+# wsgiref/typing.py doesn't exist and neither does WSGIApplication, it's a type
+# provided for type checking purposes.
+#
+# To correctly use this type stub, utilize the `TYPE_CHECKING` flag in
+# typing:
+#
+# from typing import TYPE_CHECKING
+#
+# if TYPE_CHECKING:
+#   from wsgiref import WSGIFunction
+#
```

Comment:
In the PY3 version (but not in the PY2 version) you should also tell users to use "forward reference" notation when using this (except in `# type:` comments). An example would be good.